app/src/main/kotlin/com/mapzen/erasermap/view/RouteModeView.kt

```diff
@@ -50,9 +50,6 @@ class RouteModeView : LinearLayout, RouteViewController, ViewPager.OnPageChangeL
     val mapListToggle: MapListToggleButton by lazy {
         findViewById(R.id.map_list_toggle)as MapListToggleButton
     }
-    val routeCancelButton: ImageView by lazy {
```

Comment:
Seems like it was just a mixup. We will be adding it back on the map view. Thanks for the clarification!